geode-protobuf/src/main/java/org/apache/geode/internal/protocol/protobuf/v1/ProtobufSerializationService.java

```diff
@@ -38,6 +38,9 @@ public ProtobufSerializationService() {}
   @Override
   public BasicTypes.EncodedValue encode(Object value) throws EncodingException {
     BasicTypes.EncodedValue.Builder builder = BasicTypes.EncodedValue.newBuilder();
+    if (value == null) {
+      return builder.build();
```

Comment:
I think it might be more efficient to just use EncodedValue.getDefaultInstance().  This should be the same result.

Reply:
good idea.